### ISSUE TITLE
How is optimizer_synthesizer work?

### ISSUE THREAD

**Issue by chenjunlinlin** [liangchen527/SLADD#15]
First, thanks for your work. but I want to know  how is optimizer_synthesizer work. I can't understand the loss of synthesizer. 
Can someone explain the following code. At around line 166 of the whole_model.py file. "synthesizer_loss = score_loss - (1e-5) * entropy_penalty"
